opendbc/car/honda/interface.py

```diff
@@ -69,6 +69,9 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, alpha_lo
     if 0x1C2 in fingerprint[CAN.pt]:
       ret.flags |= HondaFlags.HAS_EPB.value
 
+    if candidate in HONDA_BOSCH_CANFD and 0x184 not in fingerprint[CAN.pt]:
+      ret.flags |= HondaFlags.BOSCH_ALT_BRAKE.value
+
```

Comment:
@mvl-boston do you know what's going on here? 0x184 exists on the bus, but is undefined in the DBC, it's not the alt brake message 0x1BE. If there are CAN-FD cars that need it, I would just add `*HONDA_BOSCH_CANFD` to the 0x1BE existence check, but I don't know if there was a good reason behind this 0x184 detection.

https://github.com/commaai/opendbc/blob/28a16218639ff1a5a2bd3e36e750a52bc7f968b1/opendbc/car/honda/interface.py#L209-L212